### ISSUE TITLE
Posters obstructing #-Z quick access

### ISSUE THREAD

**Issue by broe7788** [jellyfin/jellyfin-tizen#266]
I would like to fix this. I recently updated to the latest jellyfin build. The #-z list was on the right side before the update but for some reason after the update, it shifted to the other side and is now obstructed by the movie posters. 
![PXL_20240601_172633351](https://github.com/jellyfin/jellyfin-tizen/assets/24998225/925a229c-7947-464e-b546-f87ec1b85470)

**Comment by dmitrylyzo** [comment]
Try disabling custom CSS first.

**Comment by thornbill** [comment]
Yeah those should be hidden on TVs due to the difficulty they cause when navigating with a remote.

**Comment by broe7788** [comment]
This is what I have. I don't believe any CSS is active
![PXL_20240601_180328660](https://github.com/jellyfin/jellyfin-tizen/assets/24998225/6b3385a0-b032-4695-b63f-ab05db4dcb37)

**Comment by dmitrylyzo** [comment]
> Yeah those should be hidden on TVs due to the difficulty they cause when navigating with a remote.

AlphaPicker can be used on TV when the page has enough padding on the corresponding side.

I see no padding in the screenshot at all.

**Comment by broe7788** [comment]
The padding definitely went away with the update. IDK how to work around it but as you can see, those letters hug the edge pixels HARD lol. OR actually, now that I'm thinking about it, I think the padding mightve went nonexistent on the left side and doubled on the right? I'm not sure 
![PXL_20240601_183611563](https://github.com/jellyfin/jellyfin-tizen/assets/24998225/bd8c6d87-87a1-4094-b61f-441846cc52ca)

**Comment by zhen-zen** [comment]
The issue is mainly introduced by https://github.com/jellyfin/jellyfin-web/pull/3676.

To compute the paddings, that PR uses [`max()` CSS function](https://developer.mozilla.org/en-US/docs/Web/CSS/max), which is only available to Chrome 79+.

The fallback lines above the `max()` lines didn't work with M76 Chromium in model year 2021 Tizen 6.0 TVs. The computed result in devtools shows `max()` is being used, but the output is 0px. Meanwhile, M69 Chromium in model year 2020/Tizen 5.5 would discard the property and use the fallback line.

I'm not sure if there's a way to workaround this browser compatibility issue.

**Comment by kraftwerk2020** [comment]
I applied the above CSS fix and it fixed the layout, but if I navigate to the #-z list on the left with the remote I can not navigate out of it.
